content/projects/kotlin/project-3/create-a-fragment/android-trivia-starter/README.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
This one I made a change as suggested by the linter but not sure if this is the bestw way to do it.

What is the reason for android-trivia-starter to be here like this? Should it not be a download?

Also there is not _index.md here